app/views/users/new.html.erb

```diff
@@ -0,0 +1,49 @@
+<%
+=begin%>
+<h1>User Registration</h1>
+<form action='/users' method='POST'> <br>
+  <label for='username'>Username</label> <br>
+  <input type='text' id='username' name='user[username]'> <br>
+  <label for='email'>Email</label> <br>
+  <input type='text' id='email' name='user[email]'> <br>
+  <label for='password'>Password</label> <br>
```

Comment:
This should have the user object. More like => `<label for='user[password]'> Password </label>`